src/main-enhancer.js

```diff
@@ -33,12 +33,21 @@ module.exports = overrides => storeCreator => (reducer, initialState) => {
 
   // Cannot delete data, as events could still be sent after close
   // events when a BrowserWindow is created using remote
-  let unregisterRenderer = (webContentsId) => {
+  let unregisterRenderer = webContentsId => {
     clients[webContentsId] = { active: false };
   };
 
   ipcMain.on(`${globalName}-register-renderer`, ({ sender }, { filter, clientId }) => {
     let webContentsId = sender.getId();
+
+    // if webviews are reloaded, the contents id changes
```

Comment:
```suggestion
    // If webviews are reloaded, the contents id changes
```

Reply:
Excuse me, there are multiple webviews in my project that have been restarted and then expired. How to get out of multiple webviews?